mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt

```diff
@@ -106,7 +106,7 @@ class WebViewFragment : Fragment(), ConnectionFactory.UpdateListener {
         return false
     }
 
-    private fun loadWebsite(urlToLoad: String = urltoLoad) {
+    private fun loadWebsite(urlToLoad: String = this.urlToLoad) {
```

Comment:
`this.` needed?

Reply:
Yes, previously it was `urlToLoad` (fun) vs `urltoLoad` (class).